src/inmanta/server/services/orchestrationservice.py

```diff
@@ -687,8 +687,6 @@ def safe_get(input: object, key: str, default: object) -> object:
 
         LOGGER.debug("Successfully stored version %d", version)
 
-        self.resource_service.clear_env_cache(env)
-
```

Comment:
Don't we still have to clear the cache somewhere ?

Reply:
No. The cache is cleared implicitly in the `release_version()` method, because the version number used there is always higher than the currently deployed version. So `release_version` will always have a cache miss on the increment cache, but we never have to clear the cache explicitly.